src/sentry/runner/commands/configoptions.py

```diff
@@ -209,44 +239,53 @@ def sync(ctx):
     all_options = options.filter(options.FLAG_AUTOMATOR_MODIFIABLE)
 
     options_to_update = ctx.obj["options_to_update"]
+    invalid_options = ctx.obj["invalid_options"]
     drift_found = bool(ctx.obj["drifted_options"])
     for opt in all_options:
-        if opt.name in options_to_update:
-            try:
-                _attempt_update(
-                    opt.name,
-                    options_to_update[opt.name],
-                    ctx.obj["drifted_options"],
-                    dry_run,
-                    bool(ctx.obj["hide_drift"]),
-                )
-            except Exception:
-                metrics.incr(
-                    "options_automator.run",
-                    tags={"status": "update_failed"},
-                    sample_rate=1.0,
-                )
-                raise
-        else:
-            if options.isset(opt.name):
-                if options.get_last_update_channel(opt.name) == options.UpdateChannel.AUTOMATOR:
-                    if not dry_run:
-                        try:
-                            options.delete(opt.name)
-                        except Exception:
-                            metrics.incr(
-                                "options_automator.run",
-                                tags={"status": "update_failed"},
-                                sample_rate=1.0,
-                            )
-                            raise
-                    click.echo(UNSET_MSG % opt.name)
-                else:
-                    click.echo(DRIFT_MSG % opt.name)
-                    drift_found = True
+        if opt.name not in invalid_options:
```

Comment:
This requires some refactoring. There is no reason to pass the list of drifted options to the attempt_update method anymore as we do not make all the other scenario crash upfront.

I did not do that in this PR because I know @kneeyo1 is also making important changes to this file for the slack integration so I will hold the refactoring off till he is done to avoid major conflicts to solve.